pkg/database/testing.go

```diff
@@ -154,3 +154,22 @@ func GetSessionFromTestingDB() *proto.Session {
 		HdfsUser:     "",
 		HdfsPass:     ""}
 }
+
+// MockURL returns a valid database connection URL.
+func MockURL() string {
```

Comment:
Why need a `MockURL` function, when using it, we may not know what database type we are connecting to.

Reply:
When I go through the codebase, I noticed some unit tests that just need a mockup URL. So I simply added the function database.MockURL().  You are right - we should check if we can remove it later after some code changes.